tests/e2e/run-scanner-v4.bats

```diff
@@ -340,6 +339,80 @@ teardown_file() {
     verify_scannerV4_indexer_deployed "${CUSTOM_SENSOR_NAMESPACE}"
 }
 
+@test "[Operator] Upgrade multi-namespace installation" {
+    if [[ "${ORCHESTRATOR_FLAVOR:-}" != "openshift" ]]; then
+        skip "This test is currently only supported on OpenShift"
+    fi
+    if [[ "${ENABLE_OPERATOR_TESTS:-}" != "true" ]]; then
+        skip "Operator tests disabled. Set ENABLE_OPERATOR_TESTS=true to enable them."
+    fi
+
+    # shellcheck disable=SC2030,SC2031
+    export DEPLOY_STACKROX_VIA_OPERATOR="true"
+    # shellcheck disable=SC2030,SC2031
+    export SENSOR_SCANNER_SUPPORT=true
+
+    # Install old version of the operator & deploy StackRox.
+    (
+      VERSION="${OPERATOR_VERSION_TAG}" make -C operator deploy-previous-via-olm
+      #  cd operator
+      #  ./hack/olm-operator-install.sh stackrox-operator quay.io/rhacs-eng/stackrox-operator 4.3.0 4.3.0
+    )
+    ROX_SCANNER_V4="false" _deploy_stackrox "" "${CUSTOM_CENTRAL_NAMESPACE}" "${CUSTOM_SENSOR_NAMESPACE}"
+
+    verify_scannerV2_deployed "${CUSTOM_CENTRAL_NAMESPACE}"
+    verify_scannerV2_deployed "${CUSTOM_SENSOR_NAMESPACE}"
+
+    # Upgrade operator
+    info "Upgrading StackRox Operator to version ${OPERATOR_VERSION_TAG}..."
+    VERSION="${OPERATOR_VERSION_TAG}" make -C operator upgrade-via-olm
+    info "Waiting for rhacs-operator pods to be ready"
+    "${ORCH_CMD}" -n stackrox-operator wait  --for=condition=Ready pods -l app=rhacs-operator
+
+    verify_scannerV2_deployed "${CUSTOM_CENTRAL_NAMESPACE}"
+    verify_scannerV2_deployed "${CUSTOM_SENSOR_NAMESPACE}"
+    verify_no_scannerV4_deployed "${CUSTOM_CENTRAL_NAMESPACE}"
+    verify_no_scannerV4_indexer_deployed "${CUSTOM_SENSOR_NAMESPACE}"
+
+    # Wait until ValidationWebhook is completely functional.
```

Comment:
My idea here was to separate the waiting step from the actual test logic, to ease, for example, future test refactorings.

(I used the environment variable because I had to provide a non-empty patch so that the validation hook is actually tested.)

How about I put this waiting code into a function and call it here?